widgets/src/test/java/org/pentaho/gwt/widgets/client/listbox/CustomListBoxTest.java

```diff
@@ -234,7 +234,6 @@ public void testSetSelectedItem() throws Exception {
     when( customListBox.items.indexOf( item ) ).thenReturn( selectionIndex );
     customListBox.setSelectedItem( item );
     verify( selectedItem ).onDeselect();
-    verify( customListBox ).togglePopup();
```

Comment:
A test already exists for the `itemSelected( . )` method. Maybe you could add this assertion there, or create a new test of not appropriate.